src/Agent.Listener/Configuration/ServiceControlManager.cs

```diff
@@ -53,6 +53,18 @@ public void CalculateServiceName(AgentSettings settings, string serviceNamePatte
             }
 
             serviceName = StringUtil.Format(serviceNamePattern, accountName, settings.PoolName, settings.AgentName);
+
+            if (serviceName.Length > 80)
```

Comment:
i assume 80 won't break, only 81+ will break.

Reply:
Correct. I tried it out with exactly 80 characters, and it works.